data/brands/amenity/fuel.json

```diff
@@ -2271,19 +2271,6 @@
         "name": "Hy-Vee Gas"
       }
     },
-    {
-      "displayName": "IDEMITSU",
-      "id": "idemitsu-16b3c2",
-      "locationSet": {
-        "include": ["001"],
-        "exclude": ["pl"]
```

Comment:
Is there need to remove?  I guess we can add `jp` to exclude list and complete the global one with wikidata/wikipedia